Bug Report: kafka-state-detector env not in Helm Chart

### Expected Behavior

by default kafka-state-detector should have activity and state enabled.

### Current Behavior

Both are not accessible in the Helm chart and are not set at all

### Relevant log output

```shell
{"level":"info","ts":1651076549.4356902,"caller":"kafka-state-detector/main.go:34","msg":"This is kafka-state-detector build date: 2022-04-27 16:11:40"}
{"level":"info","ts":1651076549.435948,"caller":"kafka-state-detector/main.go:51","msg":"Allowed memory size is 1073741824"}
panic: No activity or anomaly detection enabled

goroutine 1 [running]:
main.main()
	/build/cmd/kafka-state-detector/main.go:101 +0x39b
```


### Steps to Reproduce

see above

### Context (Environment)

Venice testing
prerelease7

### Possible Solution

_No response_

### Version

v0.9.x

### Anything else?

_No response_

--- a/deployment/united-manufacturing-hub/templates/kafka-state-detector.yaml
+++ b/deployment/united-manufacturing-hub/templates/kafka-state-detector.yaml
@@ -48,6 +48,12 @@ spec:
                 fieldRef:
                   fieldPath: metadata.name
 
+            - name: ACTIVITY_ENABLED
+              value: "{{.Values.kafkastatedetector.activityEnabled}}"
+
+            - name: ANOMALY_ENABLED
+              value: "{{.Values.kafkastatedetector.anomalyEnabled}}"
+
           resources: {}
       serviceAccountName: ""
       restartPolicy: Always

--- a/deployment/united-manufacturing-hub/values.yaml
+++ b/deployment/united-manufacturing-hub/values.yaml
@@ -212,6 +212,8 @@ kafkastatedetector:
   image:
     repository: unitedmanufacturinghub/kafkastatedetector
     pullPolicy: IfNotPresent
+  activityEnabled: true
+  anomalyEnabled: true
 
 ### mqttbridge ###
 mqttbridge:

--- a/golang/cmd/kafka-state-detector/main.go
+++ b/golang/cmd/kafka-state-detector/main.go
@@ -76,6 +76,7 @@ func main() {
 		go internal.StartEventHandler("[AC]", activityEventChannel, ActivityPutBackChannel)
 		go startActivityProcessor()
 	}
+	AnomalyEnabled = os.Getenv("ANOMALY_ENABLED") == "true"
 
 	if AnomalyEnabled {
 		SetupAnomalyKafka(kafka.ConfigMap{

